sdk/communication/azure-communication-networktraversal/azure/communication/networktraversal/__init__.py

```diff
@@ -6,4 +6,4 @@
 
 # pylint: disable=bad-option-value, unused-import
 from ._communication_relay_client import CommunicationRelayClient
-from ._generated.models import RouteType
+from ._generated.models import RouteType, CommunicationRelayConfiguration, CommunicationIceServer, CommunicationRelayConfigurationRequest
```

Comment:
I believe the `Request` object is internal and shouldn't be added to the public API here.